tutorials/graphql/intro-graphql/tutorial-site/content/graphql-mutations.md

```diff
@@ -72,17 +101,44 @@ mutation {
 }
 ```
 
+Let's analyze the response returned by this mutation!
+
+### Mutation Response {#mutation-response}
+
+Running the above mutation returns the following response:
+
+```graphql
+{
+  "data": {
+    "insert_todos": {
+      "returning": [
+        {
+          "id": 55386,
+          "title": "New Todo",
+          "is_completed": false,
+          "is_public": false,
+          "created_at": "2022-02-24T10:26:30.718681+00:00"
+        }
+      ]
+    }
+  }
+}
+```
+
+The mutation returns all the todo data we requested; nothing more, nothing less.
+
+You might ask - both GraphQL Mutations and Queries can return data? Yes, but there is a caveat. A GraphQL Mutation can only return the newly inserted data. A GraphQL Query can return multiple records.
```

Comment:
This is actually not true. It totally depends on the server implementation. For example, even in Hasura, we can request relationships in the response data which might be old rows.

Can we update this section to reflect that?

Reply:
Thanks for pointing it out.

I will remove the paragraph altogether to avoid confusion!